Foundation/NSOperation.swift

```diff
@@ -141,7 +141,7 @@ public class NSOperation : NSObject {
     public var threadPriority: Double = 0.5
     
     /// - Note: Quality of service is not directly supported here since there are not qos class promotions available outside of darwin targets.
-    public var qualityOfService: NSQualityOfService = .Default
+    public var qualityOfService: NSQualityOfService = .`default`
```

Comment:
Hm, I thought an enhancement to the compiler was going to fix this ugly backtick case for enum values.